net-tools/netwatch/README.md

```diff
@@ -0,0 +1,7 @@
+# Netwatch
+
+`netwatch` is a cross-platform library for monitoring of networking interfaces
+and route changes.
+
+Used in [iroh](https://github.com/n0-computer/iroh), created with love by the
+[n0 team](https://n0.computer/).
```

Comment:
please add the license footer we have in our other readmes

Reply:
done in 5e1e3e1fe